doc/api.rst

```diff
@@ -13,7 +15,7 @@ Functions
 
 .. function:: add_runs(filename: str, result)
 
-   Append a :class:`Benchmark` or :class:`BenchmarkSuite` to an existing
+   Append a :class:`pyperf.Benchmark` or :class:`pyperf.BenchmarkSuite` to an existing
```

Comment:
Since you declared `.. module:: pyperf`, is it really needed to repeat `pyperf.` prefix here?